dev-packages/node-integration-tests/suites/tracing/requests/fetch-breadcrumbs/test.ts

```diff
@@ -72,7 +72,7 @@ conditionalTest({ min: 18 })('outgoing fetch', () => {
               },
             },
           })
-          .start(done);
+          .start(closeTestServer);
```

Comment:
the main change, when we use `createTestServer`, make sure to use this instead of `done` (which under the hood also calls `done`), to make sure everything is closed properly.